client/blocks/credit-card-form/style.scss

```diff
@@ -15,7 +15,6 @@
 
 	p {
 		font-size: 12px;
-		font-weight: 100;
```

Comment:
I thought the 100 font weight was out of place here. We don't declare a thin font weight anywhere else on small text.